src/main/java/org/gridsuite/casevalidation/server/LoadFlowCaseValidationReport.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.casevalidation.server;
+
+/**
+ * @author Etienne Homer <etienne.homer at rte-france.com>
+ */
+public class LoadFlowCaseValidationReport {
+
+    private boolean ok;
```

Comment:
Why do you keep this? This is redundant with status.

Reply:
done